docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Reduce database size using Bonsai Tries
```

Comment:
```suggestion
title: Reduce database size for Bonsai Tries
```

Not sure my suggestion is best but the original title sounds to me like "if you enable bonsai tries, you will reduce database size" (which is also true, but different to this exact feature.
For this feature it's more like, "if you are using bonsai trie, enable/run this to reduce database size".